src/FunctionApps/python/GenerateCSVs/elr.py

```diff
@@ -1,8 +1,45 @@
-from typing import List, Any
-from GenerateCSVs.patient import PATIENT_COLUMNS
+from typing import List
+from GenerateCSVs.patient import PATIENT_COLUMNS, parse_patient_resource
 
-ELR_COLUMNS = PATIENT_COLUMNS + []
 
+ELR_COLUMNS = PATIENT_COLUMNS + ["loincCode", "result", "effectiveDateTime"]
 
-def elr_to_csv(bundle: dict) -> List[Any]:
-    pass
+
+def extract_loinc_lab(observation: dict) -> List[str]:
+    """
+    Given an observation FHIR profile, determine whether the profile
+    contains information related to a laboratory test and result.
+    If yes, collect the relevant lab code, test result, and test
+    date into a list.
+    """
+    code = observation["code"]["coding"][0]
+    if "loinc" in code["system"] and "valueCodeableConcept" in observation:
+        obs_date = observation["effectiveDateTime"]
+        result = observation["valueCodeableConcept"]["coding"][0]["display"]
+        return [code["code"], result, obs_date]
+    return []
+
+
+def elr_to_csv(bundle: dict) -> List[List[str]]:
+    """
+    Given a FHIR bundle containing a patient resource and one or more
+    ELR lab observations, identify the labs related to COVID and turn
+    them into patient-identified rows for use in a CSV.
+    """
+    rows_to_write = []
+    patient_info_list = []
+    for resource in bundle["entry"]:
+        if resource["resource"]["resourceType"] == "Patient":
+            patient_info_list.extend(parse_patient_resource(resource))
+        elif resource["resource"]["resourceType"] == "Observation":
+
+            # We only care about observations related to covid testing
```

Comment:
Technically we are grabbing any test not just COVID, so this comment isn't quite right. Super nit picky...